### PR TITLE
docs(templates): add the issue report & PR templates

### PR DESCRIPTION
**Pull request by luke-belton**
## Goal

Adds Support-related content for go-live.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+3. Scroll down to '....'
+4. See error
+
+### Environment
+* Bugsnag version:
+* Dart version:
+* Flutter version:
+* Flutter app or native app with flutter module:
+* iOS/Android version:
+* Simulator/emulator or physical device:
```
Comment (tomlongridge) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]: @lemnik – can you give this list the once-over and check you're happy with what we're asking for?

Wasn't sure about Dart version - is that known given the Flutter version?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+### Environment
+* Bugsnag version:
+* Dart version:
+* Flutter version:
+* Flutter app or native app with flutter module:
```
Comment (tomlongridge) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]:
```suggestion
* Flutter-first app or native app with Flutter component(s):
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (lemnik) [reply .github/ISSUE_TEMPLATE/bug_report.md]: It looks good except for that "Dart version" - that is defined by the Flutter SDK version as @tomlongridge suggested

File: .github/support.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+Or, if you wish you can [contact us directly](mailto:support@bugsnag.com) for assistance on integrating Bugsnag into your application, troubleshooting an issue or a question about our supported features.
+
+When contacting support, please include as much information as necessary, including:
+
+- example code snippet
+- steps to reproduce
+- expected/actual behaviour 
+
+* Bugsnag version:
+* Dart version:
```
Comment (lemnik) [inline .github/support.md:13]: Same as above - the Dart version is defined by which version of the Flutter SDK is being used

File: .github/support.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+When contacting support, please include as much information as necessary, including:
+
+- example code snippet
+- steps to reproduce
+- expected/actual behaviour
+
+* Bugsnag version:
+* Flutter version:
+* Flutter app or native app with flutter module:
```
Comment (tomlongridge) [inline .github/support.md:15]:
```suggestion
* Flutter-first app or native app with Flutter component(s):
```

File: .github/support.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+Or, if you wish you can [contact us directly](mailto:support@bugsnag.com) for assistance on integrating Bugsnag into your application, troubleshooting an issue or a question about our supported features.
+
+When contacting support, please include as much information as necessary, including:
+
+- example code snippet
+- steps to reproduce
+- expected/actual behaviour
+
+* Bugsnag version:
```
Comment (tomlongridge) [inline .github/support.md:13]: nit: you changed the `*`'s to `-`'s everywhere else

**Review by tomlongridge — APPROVED** [review]

**Pull request merged**